x-pack/plugins/inference/common/output/index.ts

```diff
@@ -52,18 +54,21 @@ export type OutputAPI = <
     system?: string;
     input: string;
     schema?: TOutputSchema;
+    messages?: Message[];
```

Comment:
NIT: at least add to the TSDoc, without looking at the implem I have no idea what this is compared to `input`, or why it doesn't supersede it.

Reply:
cheers, will update, WDYT about renaming it to `prevMessages`?

Reply:
renamed to `previousMessages`, LMK if that breaks your heart